data/borderlines.json

```diff
@@ -197,6 +212,21 @@
           "name": "Everest Fun Park & Resort Pvt. Ltd.",
           "latitude": 26.6431722,
           "longitude": 87.6181256
+        },
+        {
+          "name": "Urlabari Multiple Campus",
+          "longitude": 26.6651,
+          "latitude": 87.6200
+        },
+        {
+          "name": "Buddhasubba Temple",
```

Comment:
Neither is budhasubba temple

Reply:
And this, I've modified to Bhandari Krishi Farm.